.nvmrc.txt

```diff
@@ -0,0 +1 @@
+v10.16.0
```

Comment:
the name of this file, should be only `.nvmrc`

Reply:
Hi Holmes. I updated the name file. Please, You approved it